provider/views.py

```diff
@@ -261,12 +263,22 @@ def handle(self, request, post_data=None):
 
         authorization_form = self.get_authorization_form(request, client,
             post_data, data)
-
-        if not authorization_form.is_bound or not authorization_form.is_valid():
-            return self.render_to_response({
-                'client': client,
-                'form': authorization_form,
-                'oauth_data': data, })
+        
+        if data.get('response_type', None):
+            if not authorization_form.is_bound \
+                or not authorization_form.is_valid():
+                return self.render_to_response({
+                                                'client': client,
+                                                'form': authorization_form,
+                                                'oauth_data': data, })
+
+            if data.get('response_type') == 'token':
+                #uses request.user as the urls.py already required login_required
+                atm = AccessTokenModel.objects.create(user=request.user,
+                                                client=client,
+                                                scope=data.get('scope'))
```

Comment:
When the authorization request omits the 'scope' parameter, this causes an empty `scope=` to be returned in the response, which is technically incorrect in terms of [the rfc](https://tools.ietf.org/html/rfc6749#section-3.3) because it implies that an "empty" scope has been granted.  I think only including 'scope' when it was specified in the request would be a reasonable fix.